experiments/functional/backup_and_restore/setup_dependency.yml

```diff
@@ -212,6 +233,10 @@
       failed_when: "'created' not in volume_snapshot_location.stdout"
       when: lookup('env','LOCAL_SNAPSHOT') == "true"
 
+    - name: Applying Backup storage location
```

Comment:
Instead of configuring profile for backupstoragelocation we can mention it in task `Installing velero server inside cluster`.
Command will be :
```bash

           velero install \
             --provider aws \
             --bucket velero \
             --secret-file ./credentials-velero \
             --plugins velero/velero-plugin-for-aws:v1.0.0 \
             --use-volume-snapshots=false \
             --use-restic \
             --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000,profile=profile-value
```

Reply:
For this we need to create separate new block for velero install command based on profile passed. Current change is not altering the existing velero installation rather creating a backup-storage-location based on new profile.